docs/source/notebooks/time_series_analysis.ipynb

```diff
@@ -537,18 +2113,21 @@
       "text/plain": [
        "<IPython.core.display.Image object>"
       ]
      },
-     "execution_count": 11,
+     "execution_count": 14,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "resp = wps.graph_fit(ts=ts, params=pa_fn)\n",
+    "url = os.environ.get(\"WPS_URL\", \"https://pavics.ouranos.ca/twitcher/ows/proxy/finch/wps\")\n",
```

Comment:
Should this one be called `FINCH_WPS_URL` ?

Reply:
Nope, that's the raven process. It could be renamed to RAVEN_WPS_URL to make it clear, but this would require changing the Makefile.

Reply:
No if it's raven don't change to `RAVEN_WPS_URL`. But if it's really raven, well I see `finch` in the text of the url.  Is that a typo?  It's because I see `finch` in the url that I thought the variable should have been `FINCH_WPS_URL`.

Reply:
That's a mistake. I'm dense, thanks!